### PR TITLE
added blogpost to database

### PR DESCRIPTION
**Pull request by eda90**

### CODE REVIEW

**Review by LaraKiyicioglu — APPROVED** [review]

File: packages/idealo.design/src/server/db_Configuration/7-db.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+UPDATE blogposts SET previouspost where id=5 = 'contract-testing';
```
Comment (MuellerBettina) [inline packages/idealo.design/src/server/db_Configuration/7-db.sql:1]: Are you sure this line works like this? I think you might want to change it ;-)

**Review by MuellerBettina — COMMENTED** [review]
Nice Blogpost ;-)

File: packages/idealo.design/src/server/db_Configuration/7-db.sql (hunk 1 above)
Comment (LaraKiyicioglu) [reply packages/idealo.design/src/server/db_Configuration/7-db.sql:1]: Thanks for the suggestion, but works fine after using the new start command for the server ;)

Comment (nforgerit) [reply packages/idealo.design/src/server/db_Configuration/7-db.sql:1]: Well @LaraKiyicioglu there's a typo/leftover. In the last part, it is saying `... where id=5='contract-testing'`. I'm curious if it really works and if so how. :P

**Review by MuellerBettina — APPROVED** [review]

**Review by LaraKiyicioglu — APPROVED** [review]

**Pull request merged**